addons/event_sale/models/sale_order_line.py

```diff
@@ -70,6 +76,22 @@ def _compute_name(self):
         """
         super()._compute_name()
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        """ Don't create sale order line that needs an event and event ticket. We do this to avoid
+        raising the error linked to the constrain as we don't want to block the user flow when he
+        clicks on "Configure Event" to create his first event. """
+        so_lines_to_keep = []
+        for values in vals_list:
+            product = self.env['product.product'].browse(values.get('product_id'))
+            if product and product.detailed_type == 'event':
+                if values.get('event_ticket_id') and values.get('event_id'):
+                    so_lines_to_keep.append(values)
+            else:
+                so_lines_to_keep.append(values)
+
+        return super().create(so_lines_to_keep)
```

Comment:
Seems very strange, we usually don't do that within Odoo, could quickly lead to issues.

I see the feedback from the pad, maybe just adding a `target="_blank"` on the "Configure Events & Tickets" link will be sufficient, as it will allow the end user to configure tickets in a separate tab, then close and re-open his SO line configuration to see the tickets appear ?

Reply:
Indeed, I'm going for the blank option then. It's not even a flow that should happen often so it's ok and indeed avoid further complications